src/src/com/microsoft/aad/adal/DateTimeAdapter.java

```diff
@@ -50,12 +50,31 @@
             DateFormat.DEFAULT);
 
     private final DateFormat iso8601Format = buildIso8601Format();
+    private final DateFormat enUs24HourFormat = buildEnUs24HourDateFormat();
+    private final DateFormat local24HourFormat = buildLocal24HourDateFormat();
 
     private static DateFormat buildIso8601Format() {
         DateFormat iso8601Format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.US);
```

Comment:
It's not app team actually. Broker sdk is part of wpj library, Satheesh is already aware of it. And we're fixing it.